bundles/org.openhab.binding.bloomsky/src/main/java/org/openhab/binding/bloomsky/internal/dto/BloomSkyJsonSensorData.java

```diff
@@ -0,0 +1,299 @@
+/**
+ * Copyright (c) 2010-2021 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.bloomsky.internal.dto;
+
+import java.util.List;
+
+import com.google.gson.annotations.Expose;
+import com.google.gson.annotations.SerializedName;
+
+/**
+ * The {@link BloomSkyJsonSensorData} is the Java class used to map the JSON response to an BloomSky API request.
+ *
+ * @author dschoepel - Initial contribution
+ *
+ */
+public class BloomSkyJsonSensorData {
+    /* --------- Device information follows, then sensor data ------------------ */
+    @SerializedName("UTC")
+    @Expose
```

Comment:
As your binding is read-only, I'm not sure `@Expose` being of any interest.

Reply:
Removed for all but one.  Needed it to rename a not standard json variable used by the API.